java/src/main/java/com/powsybl/python/network/Dataframes.java

```diff
@@ -348,6 +357,22 @@ private static DataframeMapper<VoltageLevel> createBusBreakerViewBuses() {
                 .build();
     }
 
+    private static List<Pair<String, ConnectablePosition.Feeder>> getPositions(Map<String, List<ConnectablePosition.Feeder>> map) {
+        Set<Map.Entry<String, List<ConnectablePosition.Feeder>>> entriesSet = map.entrySet();
+        List<Pair<String, ConnectablePosition.Feeder>> entriesList = new ArrayList<>(entriesSet.size());
+        entriesSet.forEach(e -> e.getValue().forEach(feeder -> entriesList.add(Pair.of(e.getKey(), feeder))));
+        return entriesList;
+    }
+
+    private static DataframeMapper<Map<String, List<ConnectablePosition.Feeder>>> createFeederMapDataframe() {
+        return new DataframeMapperBuilder<Map<String, List<ConnectablePosition.Feeder>>, Pair<String, ConnectablePosition.Feeder>>()
+                .itemsProvider(Dataframes::getPositions)
+                .stringsIndex("connectable_id", Pair::getLeft)
+                .ints("order_position", pair -> pair.getRight().getOrder().orElse(null))
```

Comment:
Here actually the null in case there's no order is wrong, it raises an exception when the dataframe is created. I don't know yet how to fix it.